src/elements/content-sidebar/hooks/useSidebarMetadataFetcher.ts

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
+import getProp from 'lodash/get';
+import { type MessageDescriptor } from 'react-intl';
+import API from '../../../api';
+import { type ElementsXhrError, type ErrorContextProps } from '../../../common/types/api';
+import { isUserCorrectableError } from '../../../utils/error';
+import {
+    FIELD_IS_EXTERNALLY_OWNED,
+    FIELD_PERMISSIONS,
+    FIELD_PERMISSIONS_CAN_UPLOAD,
+    IS_ERROR_DISPLAYED,
+} from '../../../constants';
+
+import messages from '../../common/messages';
+
+import { type BoxItem } from '../../../common/types/core';
+import { MetadataTemplate, type MetadataEditor } from '../../../common/types/metadata';
+import { type ExternalProps } from '../MetadataSidebarRedesign';
+
+enum Status {
+    Idle = 'idle',
+    Loading = 'loading',
+    Error = 'error',
+    Success = 'success',
+}
```

Comment:
all of this should be capitalized, since this is considered constants

Reply:
done